thunder/images/images.py

```diff
@@ -1,5 +1,7 @@
 import logging
 from numpy import ndarray, arange, amax, amin, size, asarray, random, prod
+# Name changes between Python 2 and 3
+from six.moves import xrange
```

Comment:
I can definitely see the logic in that. In choosing to implement it this way, my concern was with performance: if you use `range` everywhere, then people using Python 3 will reap the benefits of `range` returning a more performant generator object, while those in Python 2 won't get this boost. However, we're probably not actually using `range` anywhere that the object being generated is actually big enough that this is an issue, so I think see your concern for being more forward-looking as winning out. I'll switch to that solution.